sktime/transformations/tests/test_base.py

```diff
@@ -295,6 +324,11 @@ def test_panel_in_primitives_out_not_supported_fit_in_transform(backend):
     assert len(Xt) == 7
 
 
+@pytest.mark.skipif(
+    not run_test_module_changed("sktime.transformations")
+    or not _check_estimator_deps(TSFreshFeatureExtractor, severity="none"),
```

Comment:
Since we have another skipif decorator below, is this necessary?

Reply:
thats an oversight, there should be only one, or they override each other